pdc/apps/release/views.py

```diff
@@ -562,7 +570,11 @@ class ProductVersionListView(SearchView):
     allow_empty = True
     template_name = "product_version_list.html"
     context_object_name = "product_version_list"
-    paginate_by = settings.ITEMS_PER_PAGE
+
+    def get_context_data(self, **kwargs):
```

Comment:
The above `get_context_data` almost same. Could implement it in a mixin and let each view inherit the mixin? Or some other way to eliminate duplicate code.